scripts/components/api-changes-validator/test-resources/.eslintrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "rules": {
+    "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "jsdoc/require-jsdoc": "off",
+    "no-duplicate-imports": "off"
```

Comment:
why to turn them off?

Reply:
The typescript idioms we use in integration test and those generated by validator are often "hanging in the void", i.e. they're not expected to be part of a program that does something useful. Their purpose is just to trigger compiler checks.